presentations/2024-01-25_coffee-and-coding/index.qmd

```diff
@@ -0,0 +1,149 @@
+---
+title: "Coffee and Coding"
+subtitle: "{targets}"
+author: "[Jacqueline Grout](mailto:jacqueline.grout1@nhs.net) and [Matt Dray](mailto:matt.dray@nhs.net)"
+date: 2024-01-25
+date-format: "MMM D, YYYY"
+knitr:
+  opts_chunk: 
+    eval: false
+    echo: true
+format:
+  revealjs:
+    theme: [default, ../su_presentation.scss]
+    transition: none
+    chalkboard:
+      buttons: false
+    preview-links: auto
+    slide-number: false
+    auto-animate: true
+    footer: |
+      view slides at [the-strategy-unit.github.io/data_science/presentations][ds_presentations]
+    width: 1920
+    height: 1080
+editor: 
+  markdown: 
+    wrap: 120
+---
+
+## `{targets}` for analysts
+
+::: columns
+::: {.column width="50%"}
+> Tom previously presented about {targets} at a coffee and coding last March and you can revisit his presentation and learn about the reasons why you should use the package to manage your pipeline and see a simple demonstration of how to use the package.
+
+
+
+> Matt has presented previously about {targets} and making your workflows (pipelines) reproducible.
+
+
+
+> So..... if you aren't really even sure why your pipeline needs managing as an analyst or whether you actually have one (you do) then links to their presentations are at the end
+:::
+
+::: {.column width="50%"}
+![](logo.png)
+:::
+:::
+
+## Aims
+
+> In this presentation we aim to demonstrate the real-world use of {targets} in an analysis project, but first a brief
+> explanation
+
+### Without {targets} we
+
+::: incremental
+-   Write a script
+-   Execute script
+-   Make changes
+-   Go to step 2
+:::
+
+### With {targets} we will
+
+::: incremental
+-   learn how the various stages of our analysis fit together
+-   save time by only running necessary stages as we cycle through the process
+-   help future you and colleagues re-visiting the analysis - Matt says "its like a time-capsule"
+-   make Reproducible Analytical Pipelines
+:::
```

Comment:
Did you want the `###` headers to appear incrementally as well? If so, I think you have to put them in a `fragment` block, like I've shown below (because I don't think you can put headers inside an `incremental` block).

```suggestion
::: fragment
### Without {targets} we
:::

::: incremental
-   Write a script
-   Execute script
-   Make changes
-   Go to step 2
:::

::: fragment
### With {targets} we will
:::

::: incremental
-   learn how the various stages of our analysis fit together
-   save time by only running necessary stages as we cycle through the process
-   help future you and colleagues re-visiting the analysis - Matt says "its like a time-capsule"
-   make Reproducible Analytical Pipelines
:::
```